### DISCUSSION TITLE
Is there a way to combine VecSkipError and StringWithSeparator

### DISCUSSION THREAD

**Discussion by pvichivanives** [jonasbb/serde_with#773] [Q&A] [answered]
Looking for a way to combine VecSkipError and StringWithSeparator so that we could use StringWithSeparator that doesn't error with 1 bad string.
Something like: "VecSkipError<StringWithSeparator::<CommaSeparator, Object>>" doesn't work because it would then be a double Vec, same with the other way round.

**Comment by jonasbb** [comment, marked as answer]
No, there is no way to combine them like you want. Writing `VecSkipError<StringWithSeparator<...>>` is the only way, but that, as you identified, creates a `Vec<Vec<...>>`. You will have to copy the code from `StringWithSeparator` and add the error handling to that.
